components/Tabs/src/private/MDCItemBar.m

```diff
@@ -384,7 +384,10 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
 
   [self configureCell:itemCell];
   [itemCell updateWithItem:item atIndex:indexPath.item count:_items.count];
-
+    
+  BOOL shouldSelectItem = [self collectionView:collectionView shouldSelectItemAtIndexPath:indexPath];
```

Comment:
Would you please add a couple of unit tests for the new behavior? I think we should test this method using an ItemBar delegate and confirming that the cells are marked as `userInteractionEnabled = YES/NO` depending on the delegate response.

Reply:
Looks good.